miden-lib/asm/sat/note_setup.masm

```diff
@@ -0,0 +1,38 @@
+use.miden::sat::prologue
+
+# MEMORY OFFSETS
+# =================================================================================================
+
+# BOOK KEEPING
+# -------------------------------------------------------------------------------------------------
+const.CURRENT_CONSUMED_NOTE_PTR=3
```

Comment:
Question: this location holds memory pointer to the note currently being processed - right?

Reply:
Yes, that's right.